.github/workflows/ci.yml

```diff
@@ -161,38 +161,48 @@ jobs:
     needs: pre-commit
     uses: ./.github/workflows/python-package-create.yml
     with:
-      repository: beeware/${{ matrix.repo }}
-      tox-source: ${{ matrix.tox-source }}
-      tox-factors: ${{ matrix.tox-factors }}
-      distribution-path: ${{ matrix.dist-path }}
+      repository: beeware/${{ matrix.repo || matrix.name }}
       build-subdirectory: ${{ matrix.build-subdir }}
     strategy:
       fail-fast: false
       matrix:
-        repo: [ briefcase, gbulb, rubicon-objc, toga ]
+        name:
+        - beeware
+        - briefcase-automation
+        - gbulb
+        - rubicon-objc
+        - toga-core
+        - toga-android
+        - toga-chart
+        - travertino
```

Comment:
What are the extra test cases validating here? briefcase-automation and toga-android make sense - they're both variations on sub packages; but it's not clear to me what validating beeware, toga-chart and travertino is adding over the base testing of gbulb as a "simple top-level package"

If there's explicit use cases being tested, it would be worth documenting them so it's clear why these projects are part of the CI checks.

Reply:
Oh, I just added every repo currently using `python-package-create.yml`. I can trim it back down....although this workflow runs quite fast now so it definitely doesn't hit CI runtime.